parm/soca/berror/soca_hybrid_bmat.yaml

```diff
@@ -0,0 +1,92 @@
+covariance model: hybrid
+components:
+- covariance:
+    covariance model: SABER
+    saber central block:
+      saber block name: EXPLICIT_DIFFUSION
+      active variables: [tocn, socn, ssh, cicen]
+      geometry:
+        mom6_input_nml: mom_input.nml
+        fields metadata: ./fields_metadata.yaml
```

Comment:
should this be the whole path?

Reply:
We've defined it this way for all the yamls that have geometry. If we have to have a full path, we'll issue a separate pr for the change.